scripts/src/javatests/org/oppia/android/scripts/coverage/CoverageRunnerTest.kt

```diff
@@ -37,7 +37,7 @@ class CoverageRunnerTest {
   }
 
   @Test
-  fun testRunCoverage_emptyDirectory_throwsException() {
+  fun testCoverageRunner_emptyDirectory_throwsException() {
```

Comment:
The test name format that I encourage is:

``testAction_withCondition_andCondition_hasOutcome``

In this case, the action is 'runWithCoverageAsync'. I suggest renaming based on that for better clarity. It should be contextually evident that ``CoverageRunner`` is being tested here since these tests are in ``CoverageRunnerTest``.

Reply:
Ah Okay! Renamed them to: 

- testRunWithCoverageAsync_emptyDirectory_throwsException
- testRunWithCoverageAsync_invalidTestTarget_throwsException
- testRunWithCoverageAsync_validSampleTestTarget_returnsCoverageData